content/pages/equipment/hammer_dressing.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "How To: Dress a Hammer"
+description: "A guide to preparing your hammer for working metal"
+date: 2024-02-11T02:33:56Z
+tags:
+- howto
+---
+{{< notice warning >}}
+Always wear a respirator and eye protection when grinding metal
+{{< /notice >}}
+
+## Why should I do this?
+
+Dressing your hammer face will prevent unsightly marks left by the edges on the faces of most commercially produced hammers.
```

Comment:
You can also dress a hammer to make its face more or less convex, which changes how aggressively it will move material, right?